Examples/ObjectiveC/DDSLayerSelectionExample.m

```diff
@@ -0,0 +1,100 @@
+//
+//  DDSLayerSelectionExample.m
+//  Examples
+//
+//  Created by Jordan Kiley on 3/21/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+// url to use: 
+
+#import "DDSLayerSelectionExample.h"
+@import Mapbox;
+
+NSString const *MBXExampleDDSLayerSelection = @"DDSLayerSelectionExample";
+
+@interface DDSLayerSelectionExample () <MGLMapViewDelegate, UIGestureRecognizerDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation DDSLayerSelectionExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds];
+    self.mapView.delegate = self;
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(39.232253141714885,-97.91015624999999)];
+    
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+    [self.view addSubview:self.mapView];
+    
+    // Add a tap gesture recognizer to the map view.
+    UITapGestureRecognizer *gesture = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(handleTap:)];
+    gesture.delegate = self;
+    gesture.numberOfTapsRequired = 1;
+    [self.mapView addGestureRecognizer:gesture];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    // Load a tileset containing U.S. states and their population density.
+    NSURL *url = [NSURL URLWithString:@"mapbox://examples.69ytlgls"];
```

Comment:
Additionally, if this example uses datasets/tilesets, it should explain what these are and potentially address https://github.com/mapbox/ios-sdk-examples/issues/72.